### ISSUE TITLE
Support for pbutils

### ISSUE THREAD

**Issue by idcmp** [gstreamer-java/gst1-java-core#284]
I'd love to use Discoverer from Java, but afaict it's unsupported by GStreamer Java.

**Comment by neilcsmith-net** [comment]
Yes, to fully use at present you'd have to write the relevant mapping and register your GObject subclass.
